pkg/chartutil/values.go

```diff
@@ -288,6 +288,27 @@ func coalesceTables(dst, src map[string]interface{}) map[string]interface{} {
 	return dst
 }
 
+// ToRenderValues composes the struct from the data coming from the Releases, Charts and Values files
+func ToRenderValues(chrt *chart.Chart, chrtVals *chart.Config, options map[string]interface{}) (Values, error) {
```

Comment:
The free `options` map approach is very rubyish and I feel comfortable with it but I am wondering what's your take. Would you prefer me to list the arguments?